doc/release-checklist.md

```diff
@@ -22,6 +22,7 @@
 
 - [ ] Review `-h`, `--help`, and the `man` page. All of these are shown in
       the output of the CI job called *Documentation*, so look there.
+      The CI workflow corresponding to the tip of the master branch is a good place to look.
```

Comment:
Maybe add a link to https://github.com/sharkdp/bat/actions/workflows/CICD.yml?query=branch%3Amaster?

Reply:
Nice! I will add a link to that before merging

Reply:
Hmm that's a bit risky though because that query sometimes returns master branches in forks, that might be out of date:

<img width="537" alt="Skärmavbild 2022-05-09 kl  09 08 17" src="https://user-images.githubusercontent.com/115040/167358136-7d920e75-b20a-49ed-a226-dd078783f30a.png">

Personally I just go to https://github.com/sharkdp/bat and click the green checkmark next to the commit at the top.

So I think I'll go ahead and merge this PR as is.